src/utils/utilities.tsx

```diff
@@ -0,0 +1,16 @@
+export const Linkify = ({ children }) => {
+  const isUrl = (word) => {
+    const urlPattern =
+      /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/gm;
+    return word.match(urlPattern);
+  };
+
+  const addMarkup = (word) => {
+    return isUrl(word) ? `<a href="${word}">${word}</a>` : word;
+  };
+
+  const words = children.split(" ");
+  const formatedWords = words.map((w) => addMarkup(w));
+  const html = formatedWords.join(" ");
+  return <span dangerouslySetInnerHTML={{ __html: html }} />;
```

Comment:
Can I ask why you decided to set `dangerouslySetInnerHTML`? Please find an alternative as it is obvious from the attribute name that you probably shouldn't use it.

Reply:
Yeah, I didn't really understand what that was. Admittedly, I found this function online and I was able to implement it into my code. I'll look into a better alternative.